timeless/reservations/views.py

```diff
@@ -44,6 +46,31 @@ class CommentView(CrudAPIView):
     url_lookup = "comment_id"
 
 
+class JsonView(views.MethodView):
```

Comment:
@carlosmiranda I don't particularly like this name, it sounds like generic one.
Let's call it ReservationListView or smth like that